components/Blog/Blog.tsx

```diff
@@ -26,7 +26,12 @@ export const Blog: FC<IBlogProps> = ({ imgUrl, name, bio, avatarSize = 'large',
           <h2 className='m-0'>{name}</h2>
           <div className={classes['social-media-links']}>
             {links.map((link) => (
-              <a key={link.link} href={addLinkPrefix(link)} target='_blank' rel='noreferrer'>
+              <a
+                key={link.link.replace('/@', '/')}
```

Comment:
you should remove @ from link for href inside addLinkPrefix function
your changes does NOT change anything (

Reply:
addLinkPrefix di ichidagi hamma return qilingan link dan o'chirishim kerakmi.O'rtada ham @ kelib qolsa u linkdi ichiga kirib ketgan bo'lsa xato bo'lib qolmaydimi.Misol uchun user@name

Reply:
@musojon1 telegram username @ bilan boshlangan bo'lsa (@test_username, @kimdirda, @username) shu @ ni olib tashlash kerak (@username -> username)